Plot.R

```diff
@@ -113,241 +137,731 @@ parsesubject <- function(row, half){
   timesubjectindex <- row["TimeSubjectIndex"]
   strlist <- (strsplit(toString(timesubjectindex), "_"))
   
-  if(toString(half)=="2"){
+  if(toString(half) == "2"){
     return(unlist(strlist)[2])
   }
   else{
     return(as.numeric(unlist(strlist)[1]))
   }
 }
 
-############################################
-# List of times to include in 48 hour window
-############################################
-window1.begin <- as.numeric((chron("12/01/2014", format=c(dates="m/d/y")) - chron("2014-10-21",
-                                                          format=c(dates="y-m-d")) -1)*24 + 10)
-window1.end <- as.numeric((chron("12/03/2014", format=c(dates="m/d/y")) - chron("2014-10-21", 
-                                                          format=c(dates="y-m-d"))-1)*24 + 10)
-
-window2.begin <- as.numeric((chron("12/15/2014", format=c(dates="m/d/y")) - chron("2014-10-21", 
-                                                          format=c(dates="y-m-d")) -1)*24 + 10)
-
-window2.end <- as.numeric((chron("12/17/2014", format=c(dates="m/d/y")) - chron("2014-10-21", 
-                                                          format=c(dates="y-m-d"))-1)*24 + 10)
-
-TimeList<- c(seq(window1.begin, window1.end), seq(window2.begin, window2.end))
-TimeList_window1 <- c(seq(window1.begin, window1.end))
-TimeList_window2 <- c(seq(window2.begin, window2.end))
+TimeList_window1 <- c(seq(922, 1232))
+TimeList_window2 <- c(seq(1328, 1545))
 
 ###############################
 # Heatmap of variance explained
 ###############################
 
-transformed.communication.activity <- readr::read_csv("transformed.communication.activity.csv")
+transformed.communication.activity <- 
+  readr::read_csv("transformed.communication.activity.csv")
 transformed.communication.activity["X1"] = NULL
 
 # Two 48 Hour visits containing measurements of blood pressure, heart rate
 BP_HR <- readr::read_csv("heartrate.bp.csv")
-Full <- dplyr::full_join(BP_HR, transformed.communication.activity, by="TimeSubjectIndex")
+Full <- dplyr::full_join(BP_HR, transformed.communication.activity,
+                         by = "TimeSubjectIndex")
 bp.HR.com.act <- na.omit(Full)
 postscript("Variability.Act.Com.BP.eps")
 PlotR2s(bp.HR.com.act[,5:dim(bp.HR.com.act)[2]], 
-        "Variance Explained in Activity, Communication, Biometric Data (Visits 1 and 2)", "BP")
+        paste("Variance Explained in Activity,", 
```

Comment:
paste is just keeping the line under 80 characters without the next line characters that breaking them up without paste would include